examples/rpc/rpc_from_vcn_module/main.tf

```
@@ -0,0 +1,143 @@
+# Copyright (c) 2019, 2021, Oracle Corporation and/or affiliates.
+# Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl/
+
+# Version requirements
+
+terraform {
+  required_providers {
+    oci = {
+      source  = "hashicorp/oci"
+      version = ">=4.41.0"
+    }
+  }
+  required_version = ">= 1.0.0"
+}
+
+# Resources
+
+module "vcn_local" {
+  # this module use the generic vcn module and configure it to act as local vcn
+  source  = "../../../"
+  
+  # general oci parameters
+  compartment_id = var.compartment_id
+  label_prefix   = "local"
+  freeform_tags  = var.freeform_tags
+
+  # vcn parameters
+  create_drg               = true
+  create_rpc               = true           
+  create_internet_gateway  = false           
+  lockdown_default_seclist = false           
+  create_nat_gateway       = true           
+  create_service_gateway   = false           
+  vcn_cidrs                = tolist([var.local_vcn_cidr])
+  vcn_dns_label            = "vcnlocal"
+  vcn_name                 = "vcnlocal"
+  nat_gateway_route_rules = [ 
+    {
+      destination       = var.remote_vcn_cidr # set remote vcn cidr as destination cidr 
+      destination_type  = "CIDR_BLOCK"     
+      network_entity_id = "drg"            
+      description       = "Terraformed - User added Routing Rule to remote vcn through DRG"
+    },
+  ]
+  providers = {
+    oci = oci
+  }
+  
+}
+
+module "subnet_local" {
```

Please use resources instead of external modules we don't maintain directly. This is to reduce external dependencies.
```suggestion
resource "oci_core_subnet" "local" {
```